src/razor/src/blazorDebug/blazorDebugConfigurationProvider.ts

```diff
@@ -125,6 +131,13 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
         inspectUri: string,
         url: string
     ) {
+        const wasmConfig = vscode.workspace.getConfiguration('csharp');
+        const useVSDbg = configuration.useVSDbg || wasmConfig.get<boolean>('mono.debug.useVSDbg') == true;
```

Comment:
>wasmConfig.get<boolean>('mono.debug.useVSDbg') == true [](http://example.com/codeflow?start=51&length=54)

Do we need to do **something** to better handle pre-.NET 9 projects? (Assuming the new architecture doesn't handle them)

Two possible versions of something that occur to me:
* Move the user preference to C# Dev Kit, and only apply it there since the C# extension doesn't normally have a lot of information on the target framework. We could leave the debug configuration option here, but change the description.
* Change this configuration provider so it does check the target framework

Reply:
Checking it in two different places, one if has c# dev kit installed using the QueryExecutionService, and the other one is basically reading the project file and trying to detect if c# dev kit is not installed. 
If for some reason I can't get the framework it will not use vsdbg for wasm.